runsc/cmd/checkpoint.go

```diff
@@ -156,3 +159,56 @@ func (c *Checkpoint) Execute(_ context.Context, f *flag.FlagSet, args ...any) su
 
 	return subcommands.ExitSuccess
 }
+
+// CheckpointCompression represents checkpoint image writer behavior. The
+// default behavior is to compress because the default behavior used to be to
+// always compress.
+type CheckpointCompression int
+
+const (
+	// CheckpointCompressionNone.
+	CheckpointCompressionNone CheckpointCompression = 0x0
+
+	// CheckpointCompressionUse.
+	CheckpointCompressionUse CheckpointCompression = 0x1
+)
+
+func newCheckpointCompressionValue(val CheckpointCompression, p *CheckpointCompression) *CheckpointCompression {
+	*p = val
+	return (*CheckpointCompression)(p)
+}
+
+// Set implements flag.Value.
+func (g *CheckpointCompression) Set(v string) error {
+	switch v {
+	case "none":
+		*g = CheckpointCompressionNone
+	case "", "use":
```

Comment:
I suggest not using the string `"use"` to mean the current behavior... Maybe `"flate-bestspeed"`?
The intention is that it should be possible to add in a new type of compression later on (say `"zstd-fastest"`) and having the previous set of flag values still make sense relative to it.